functions/pubsub/test/index.test.js

```diff
@@ -22,15 +22,13 @@ function getSample () {
   var pubsub = {
     topic: sinon.stub().returns(topic)
   };
-  var gcloud = {
-    pubsub: sinon.stub().returns(pubsub)
-  };
+  var PubSubMock = sinon.stub().returns(pubsub);
   return {
     sample: proxyquire('../', {
-      gcloud: gcloud
+      '@google-cloud/pubsub': PubSubMock
     }),
     mocks: {
-      gcloud: gcloud,
+      PubSub: PubSubMock,
```

Comment:
Me too, I've been changing them as I see them, older samples don't meet the gold standard.